libs/chlorophyll/scss/components/form/checkbox/_mixins.scss

```diff
@@ -53,33 +56,14 @@ $invalid-color: tokens.$intent-danger-background;
     opacity: 1;
   }
 
-  /* Checked state */
+  // Checked state
   #{$selector} #{$checkbox-selector}:checked ~ i {
     background-color: var(--gds-comp-checkbox-container-color-selected);
   }
 
-  /* Valid state */
```

Comment:
Not sure how you want to do it, but couldn't we remove commented code? It's still in the git history if we want to revert it later on.

Reply:
Yes! It is removed 👍